scripts/dataset_processing/get_ami_data.py

```diff
@@ -90,35 +45,56 @@
         type=str,
         default='AMIdev_input_manifest.json',
```

Comment:
I think its better if we skip `input` keyword in these manifest files. Just have AMI_dev_manifest.json